claat/fetch/remotehttp/remotehttp_test.go

```diff
@@ -0,0 +1,67 @@
+package remotehttp
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+func TestFetch(t *testing.T) {
+	type input struct {
+		statusCode int
+	}
+	type output struct {
+		ok bool
+	}
+	tests := []struct {
+		in  input
+		out output
+	}{
+		{
+			in: input{
+				statusCode: http.StatusOK,
+			},
+			out: output{
+				ok: true,
+			},
+		},
+		{
+			in: input{
+				statusCode: http.StatusNotFound,
+			},
+			out: output{},
+		},
+	}
+	for _, tc := range tests {
+		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: line longer than 80 characters

Reply:
80 char limit isn't typically enforced in Go files